dspace-api/src/main/java/org/dspace/google/GoogleRecorderEventListener.java

```diff
@@ -40,7 +40,9 @@
  * Time: 10:05
  *
  * Notify Google Analytics of... well anything we want really.
+ * @deprecated
```

Comment:
Could we add a simple note to say "Use org.dspace.google.GoogleAsyncEventListener instead", or similar? Usually it's helpful to note what the replacement is within the deprecated class.

Reply:
Oh, wait.. I just realized you made this change in a shared branch.  So, I can add this comment myself :)